src/components/PlatformUserForm.tsx

```diff
@@ -122,7 +122,7 @@ export function PlatformUserForm({
     : [];
 
   return (
-    <Paper elevation={0} sx={{ py: 4, px: 6, borderRadius: '16px' }}>
+    <Paper elevation={0} sx={{ p: '32px', borderRadius: '16px' }}>
```

Comment:
use the number version when defining px in MUI considering that it will multiply it by 8
```suggestion
    <Paper elevation={0} sx={{ p: 4, borderRadius: '16px' }}>
```